_schemaV2.graphql

```diff
@@ -7335,6 +7343,32 @@ type DeleteBankAccountPayload {
   me: Me
 }
 
+type deleteConversationFailure {
```

Comment:
Should this type be lower-cased named? (looks like all the other types are capitalized) #minor